exercises/gigasecond/example.h

```diff
@@ -1,7 +1,9 @@
 #if !defined(GIGASECOND_H)
 #define GIGASECOND_H
 
-#include <boost/date_time/posix_time/posix_time.hpp>
```

Comment:
We really shouldn't bother removing this here. It doesn't save anything since it always must be included to use this header anyways.

Yes it might be against some companies guidelines, but we're going for simplicity here instead of strictly following some definition of best practice. These tests just aren't going to do much to help teach best practices on a large Cpp codebase.